tests/testdata/outputs_expected/kinds_of_calls.sol.o.json

```diff
@@ -1,54 +1 @@
-      "swc-id": "104",
-      "min_gas_used": 1167,
-      "max_gas_used": 35919,
-      "title": "Unchecked CALL return value",
-      "type": "Informational"
-    },
-    {
-      "address": 1038,
-      "contract": "Unknown",
-      "debug": "<DEBUG-DATA>",
-      "description": "This contract executes a message call to an address provided as a function argument. Generally, it is not recommended to call user-supplied addresses using Solidity's call() construct. Note that attackers might leverage reentrancy attacks to exploit race conditions or manipulate this contract's state.",
-      "function": "_function_0xeea4c864",
-      "swc-id": "107",
-      "min_gas_used": 477,
-      "max_gas_used": 1229,
-      "title": "Message call to external contract",
-      "type": "Warning"
-    },
-    {
-      "address": 1046,
-      "contract": "Unknown",
-      "debug": "<DEBUG-DATA>",
-      "description": "The return value of an external call is not checked. Note that execution continue even if the called contract throws.",
-      "function": "_function_0xeea4c864",
-      "swc-id": "104",
-      "min_gas_used": 1192,
-      "max_gas_used": 35944,
-      "title": "Unchecked CALL return value",
-      "type": "Informational"
-    }
-  ],
-  "success": true
-}
+{"error": null, "issues": [{"address": 626, "contract": "Unknown", "debug": "<DEBUG-DATA>", "description": "The return value of an external call is not checked. Note that execution continue even if the called contract throws.", "function": "_function_0x141f32ff", "max_gas_used": 35856, "min_gas_used": 1104, "swc-id": "104", "title": "Unchecked CALL return value", "type": "Informational"}, {"address": 857, "contract": "Unknown", "debug": "<DEBUG-DATA>", "description": "The return value of an external call is not checked. Note that execution continue even if the called contract throws.", "function": "_function_0x9b58bc26", "max_gas_used": 35913, "min_gas_used": 1161, "swc-id": "104", "title": "Unchecked CALL return value", "type": "Informational"}, {"address": 1038, "contract": "Unknown", "debug": "<DEBUG-DATA>", "description": "This contract executes a message call to an address provided as a function argument. Generally, it is not recommended to call user-supplied addresses using Solidity's call() construct. Note that attackers might leverage reentrancy attacks to exploit race conditions or manipulate this contract's state.", "function": "_function_0xeea4c864", "max_gas_used": 1223, "min_gas_used": 471, "swc-id": "107", "title": "Message call to external contract", "type": "Warning"}, {"address": 1046, "contract": "Unknown", "debug": "<DEBUG-DATA>", "description": "The return value of an external call is not checked. Note that execution continue even if the called contract throws.", "function": "_function_0xeea4c864", "max_gas_used": 35938, "min_gas_used": 1186, "swc-id": "104", "title": "Unchecked CALL return value", "type": "Informational"}], "success": true}
```

Comment:
Can we make this pretty again for readability? :slightly_smiling_face:

Reply:
https://github.com/ConsenSys/mythril-classic/pull/764/files#diff-1bd4444f8e8f9884576dd475243c55f2 😉

Reply:
Just noted. Pardon my stale notifications. :laughing: